lib/services/search.js

```diff
@@ -0,0 +1,395 @@
+'use strict';
+
+const es = require('elasticsearch'),
```

Comment:
Different const name perhaps? I see `es` and think `ecmascript`. Not a deal breaker of course.